### PR TITLE
Integrate markdownlint

### LINKED ISSUE #114: Travis: run mdl/luacheck
Via https://github.com/awesomeWM/awesome-www/pull/112#issuecomment-324276250.

### PR DESCRIPTION
**Pull request by psychon**
This fixes #114.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,51 +21,60 @@ To view it, open `html/index.html` in your web browser.
 You can simulate running a web server using Python, which will automatically
 open `index.html` when following a link to a directory:
 
-    $ cd html
-    $ python3 -m http.server -b localhost 8000 &
+    cd html ; python3 -m http.server -b localhost 8000 &
```
Comment (blueyed) [inline README.md]: `&&` instead of `;` ?

File: README.md [hunk 2]
```diff
@@ -21,51 +21,60 @@ To view it, open `html/index.html` in your web browser.
...
 
 1. Add your image to the folder images/screenshots with an appropriate name.
 
-2. Add a new `<figure>` tag in the bottom of *screenshots.mdwn*,
+1. Add a new `<figure>` tag in the bottom of *screenshots.mdwn*,
 
-3. Inside the new tag, add the screenshots with a <img> tag and use the `<figcaption>` to add caption to the image, explaining what is being used on the screenshot.
+1. Inside the new tag, add the screenshots with a `<img>` tag and use the
+   `<figcaption>` to add caption to the image, explaining what is being used on
+   the screenshot.
```
Comment (blueyed) [inline README.md]: There is a setting for this - I personally like the ordered style better.

File: README.md [hunk 3]
```diff
@@ -21,51 +21,60 @@ To view it, open `html/index.html` in your web browser.
...
-2. If everything is right and looks good, you're ready do make a Pull Request.
+1. If everything is right and looks good, you're ready do make a Pull Request.
 
 ## Publishing
 
 The master branch gets built by
 [Travis CI](https://travis-ci.org/awesomeWM/awesome-www/), and is then published
-through [Github's Organization Pages](https://github.com/awesomeWM/awesomeWM.github.io).
+through [Github's Organization
+Pages](https://github.com/awesomeWM/awesomeWM.github.io).
```
Comment (blueyed) [inline README.md]: You could wrap the whole link from `[Github's …`.

File: README.md (hunk 2 above)
Comment (blueyed) [reply README.md]: Something along this maybe?  (had this laying around from the last time)

```
% cat .markdownlint-style.rb
all

rule 'MD029', :style => "ordered"

% cat .mdlrc
style ".markdownlint-style.rb"
```

Comment (psychon) [reply README.md]: I briefly looked at it, saw that it required adding another file to the repo and thought "no!". :-/

**Comment by psychon** [comment]
Two out of three comments handled, remaining one is the separate style file (actually, two extra files just to change how ordered lists are formatted... :-( ).

**Pull request merged**